Backend/utils/db_utils.py

```diff
@@ -0,0 +1,224 @@
+import json
+import boto3
+import MentiiLogging as MentiiLogging
+from boto3.dynamodb.conditions import Key, Attr
+from botocore.exceptions import ClientError
+
+logger = MentiiLogging.getLogger()
+
+def createTableFromFile(settings_file, dbInstance):
+  try:
+    with open(settings_file) as f:
+      data = f.read()
+      settings = json.loads(data)
+
+      attribute_definitions = settings.get("AttributeDefinitions")
+      key_schema = settings.get("KeySchema")
+      provisioned_throughput = settings.get("ProvisionedThroughput")
+      table_name = settings.get("TableName")
+
+      # check for required parameters
+      if table_name is not None and key_schema is not None and attribute_definitions is not None and provisioned_throughput is not None:
+        if len(key_schema) > 0 and len(attribute_definitions) > 0:
+          table = dbInstance.create_table(
+              AttributeDefinitions=attribute_definitions,
+              KeySchema=key_schema,
+              ProvisionedThroughput=provisioned_throughput,
+              TableName=table_name
+            )
+          f.close()
+          return table
+      f.close()
+      raise(IOError)
+  except IOError as e:
+    message = "Unable to create table from file"
+    logger.exception(message + ': ' + settings_file + '\n' + str(e))
+    return message
```

Comment:
Don't return two different types. Message is a string and table is a dictionary. Any function that calls this doesn't immediately know what it is getting. It's impossible in any language that has types and that's because it's a dangerous thing.